ironblogger/templates/layout.html

```diff
@@ -26,11 +26,10 @@
 					{% set targ, text, method = ('do_logout', 'Log Out', 'post') %}
 					{% endif %}
 					<ul class="nav nav-justified">
-						<li><a href="{{ url_for('show_all_posts') }}">Posts</a></li>
```

Comment:
We should probably still have a nav item for the front page (maybe home? maybe keep calling it posts? not sure); the fact that the site's top level header is a link to the front page probably isn't obvious enough given that there's actually functionality there now.